src/github.com/matrix-org/go-neb/services/slackapi/slackapi.go

```diff
@@ -53,6 +56,20 @@ func (s *Service) OnReceiveWebhook(w http.ResponseWriter, req *http.Request, cli
 	w.WriteHeader(200)
 }
 
+// Register joins all configured rooms
+func (s *Service) Register(oldService types.Service, client *matrix.Client) error {
+	for _, mapping := range s.Hooks {
+		if _, err := client.JoinRoom(mapping.RoomID, "", ""); err != nil {
+			log.WithFields(log.Fields{
+				log.ErrorKey: err,
+				"room_id":    mapping.RoomID,
+				"user_id":    client.UserID,
+			}).Error("Failed to join room")
+		}
+	}
+	return nil
```

Comment:
Not following. The person who makes a `slackapi` needs to be given a URL they can tell their Slack apps to use. In Go-NEB, you have to use the webhook URL given to your service, no other URL will map to your service otherwise.